### PR TITLE
Roman/bugfix support bedrock embeddings

### PR DESCRIPTION
**Pull request by rbiseck3**
### Description
This PR resolved the following open issue: [bug/bedrock-encoder-not-supported-in-ingest](https://github.com/Unstructured-IO/unstructured/issues/2319). To do so, the following changes were made:
* All aws configs were added as input parameters to the CLI
* These were mapped to the bedrock embedder when an embedder is generated via `get_embedder`
* An ingest test was added to call the aws bedrock service
* Requirements for boto were bumped because the first version to introduce the bedrock runtime, which is required to hit the bedrock service, was introduced in version `1.34.63`, which was ahead of the version of boto pinned.

### CODE REVIEW

File: unstructured/ingest/cli/interfaces.py [hunk 1]
```diff
@@ -434,6 +434,24 @@ def get_cli_options() -> t.List[click.Option]:
...
+                ["--embedding-aws-access-key-id"],
+                help="AWS access key used for AWS-based embedders, such as bedrock",
+                type=str,
+                default=None,
+            ),
+            click.Option(
+                ["--embedding-aws-secret-access-key"],
+                help="AWS secret key used for AWS-based embedders, such as bedrock",
+                type=str,
+                default="us-west-2",
```
Comment (potter-potter) [inline unstructured/ingest/cli/interfaces.py:447]: should this be in --embedding-aws-region?

**Comment by potter-potter** [comment]
did you get changes in expected output when running the .sh? My embeddings are now 8 decimals instead of 17 decimals.

**Comment by potter-potter** [comment]
Would love to see this running in test_ingest_src. Can you fix the lint and shfmt.

**Comment by rbiseck3** [comment]
> test_ingest_src failed in CI

CI needs to have updated AWS creds to have permissions to hit the bedrock model.

**Comment by potter-potter** [comment]
> > test_ingest_src failed in CI
> 
> CI needs to have updated AWS creds to have permissions to hit the bedrock model.

did you want to give us-east-2 a try for the region? locally I'm able to run bedrock on both platform and command line with same old creds.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,8 @@
 * **Clarify IAM Role Requirement for GCS Platform Connectors**. The GCS Source Connector requires Storage Object Viewer and GCS Destination Connector requires Storage Object Creator IAM roles.
 * **Fix OneDrive dates with inconsistent formatting** Adds logic to conditionally support dates returned by office365 that may vary in date formatting or may be a datetime rather than a string. See previous fix for SharePoint
 * **Adds tracking for AstraDB** Adds tracking info so AstraDB can see what source called their api.
+* **Support AWS Bedrock Embeddings in ingest CLI** The configs required to instantiate the bedrock embedding class are now exposed in the api and the version of boto being used meets the minimum requirement to introduce the bedrock runtime required to hit the service.
+>>>>>>> 6a63c941c (bump changelog)
```
Comment (potter-potter) [inline CHANGELOG.md:18]: this seems strange

**Review by potter-potter — APPROVED** [review]
This looks good.

**Pull request merged**